app/views/posts/_write.haml

```diff
@@ -22,6 +23,7 @@
         = f.hidden_field :character_id, id: 'reply_character_id'
         = f.hidden_field :icon_id, id: 'reply_icon_id'
         = f.hidden_field :character_alias_id, id: 'reply_character_alias_id'
+        = f.hidden_field :editor_mode, value: editor_mode, id: 'editor_mode'
```

Comment:
Should probably do this in the controller - `@post.editor_mode = current_user.default_editor` and then here just let it handle everything magically